### ISSUE TITLE
IPI Ovirt 

### ISSUE THREAD

**Issue by mathgonzlez** [okd-project/okd#1795]
**Describe the bug**
After installing OKD (version 4.13.0-0.okd-2023-06-04-080300) using the IPI method within RHV 4.4SP1, the cluster was successfully installed.
I conducted a test by gracefully shutting down the cluster as per the documentation: [Graceful Cluster Shutdown](https://docs.openshift.com/container-platform/4.13/backup_and_restore/graceful-cluster-shutdown.html), and everything worked without any issues. 
However, when attempting to shut down the nodes forcibly, by forcefully stopping the VMs to simulate an unexpected failure, the cluster could not recover. 
While one of the workers could successfully obtain both an IP and the VIPs for Ingress and API, the other nodes displayed the following messages:

**Version**
OKD version: 4.13.0-0.okd-2023-06-04-080300
Install method: IPI
Cloud platform: RHV 4.4SP1

**How reproducible**
Always

**Log bundle**
I cant share must-gather because my cluster is unhealty

Thanks

**Comment by vrutkovs** [comment]
>the cluster could not recover.

Do masters boot?

>While one of the workers could successfully obtain both an IP and the VIPs for Ingress and API,

Why would a worker get an IP for API?

**Comment by mathgonzlez** [comment]
Sorry, it was a mistake, I meant the control planes. If the nodes boot.
![image](https://github.com/okd-project/okd/assets/57609433/68603fac-74c5-439e-ad10-6c86a70bfa65)

**Comment by vrutkovs** [comment]
>Nov 13 20:48:06 okd-ovirt-cdgkj-master-2 afterburn[852]: 3: error sending request for url (http://169.254.169.254/latest/meta-data/public-keys): error trying to connect: tcp connect error: Network is unreachable (os error 101)

afterburn is unable to reach metadata service.

**Comment by mathgonzlez** [comment]
> > Nov 13 20:48:06 okd-ovirt-cdgkj-master-2 afterburn[852]: 3: error sending request for url (http://169.254.169.254/latest/meta-data/public-keys): error trying to connect: tcp connect error: Network is unreachable (os error 101)
> 
> afterburn is unable to reach metadata service.

Yes i see that, But why this only happend when i shutdown my nodes in a not gracefully?
